src/android/toga_android/app.py

```diff
@@ -2,6 +2,7 @@
 from .window import Window
 
 
+# `MainWindow` is defined here in `app.py`, not `window.py`, to mollify the test suite.
```

Comment:
Yes - the test suite validates that every backend matches the Dummy backend in terms of declarations. The location of MainWindow is probably arguable; the case for it being in app.py is that the Main Window is the most obvious physical manifestation of the app.

Reply:
Yeah. I think this is OK!

I do think it was pretty complex for me to figure out why this test was failing. I'm proud I pulled it off. I suppose that's a good reason for me to improve the outputted failure message, so that others can be less confused in the future. I'll make a personal note to see if I can do that sometime.